src/main/java/org/zalando/nakadi/repository/db/TimelineDbRepository.java

```diff
@@ -31,6 +37,10 @@ public TimelineDbRepository(final JdbcTemplate jdbcTemplate, final ObjectMapper
         super(jdbcTemplate, jsonMapper);
     }
 
+    public List<Timeline> list() {
```

Comment:
I think it's very inconsistent that we have methods named `list` and `listTimelines` here.